### PR TITLE
Internal: Trigger Editor panels on tests [ED-14905]

### PR DESCRIPTION
**Pull request by rami-elementor**

### CODE REVIEW

File: tests/playwright/pages/editor-page.ts [hunk 1]
```diff
@@ -481,15 +481,84 @@ export default class EditorPage extends BasePage {
...
+	async hasTopBar() {
+		return await this.page.locator( EditorSelectors.topBar.wrapper ).isVisible();
+	}
+
+	/**
+	 * Open the menu panel.
+	 */
+	async openMenuPanel() {
+		await this.page.locator( EditorSelectors.panels.menu.footerButton ).click();
+		await this.page.waitForSelector( EditorSelectors.panels.menu.wrapper );
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:498]: `await this.page.locator( EditorSelectors.panels.menu.wrapper ).waitFor()`

File: tests/playwright/pages/editor-page.ts [hunk 2]
```diff
@@ -481,15 +481,84 @@ export default class EditorPage extends BasePage {
...
+	}
+
+	/**
+	 * Open the elements/widgets panel.
+	 */
+	async openElementsPanel() {
+		const hasTopBar = await this.hasTopBar();
+
+		if ( hasTopBar ) {
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:507]: `async hasTopBar()` you have a function already. Lets reuse it

File: tests/playwright/pages/editor-page.ts [hunk 3]
```diff
@@ -481,15 +481,84 @@ export default class EditorPage extends BasePage {
...
+			await this.page.locator( EditorSelectors.topBar.wrapper ).getByText( 'Elements' ).click();
+		} else {
+			await this.page.locator( EditorSelectors.panels.elements.footerButton ).click();
+		}
+
+		await this.page.waitForSelector( EditorSelectors.panels.elements.wrapper );
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:513]: same here:
await this.page.locator( EditorSelectors.panels.elements.wrapper ).waitFor()

File: tests/playwright/pages/editor-page.ts [hunk 4]
```diff
@@ -481,15 +481,84 @@ export default class EditorPage extends BasePage {
...
+	async openPageSettingsPanel() {
+		const hasTopBar = await this.hasTopBar();
+
+		if ( hasTopBar ) {
+			await this.page.locator( EditorSelectors.topBar.wrapper ).getByText( 'Page Settings' ).click();
+		} else {
+			await this.page.locator( EditorSelectors.panels.pageSettings.footerButton ).click();
+		}
+
+		await this.page.waitForSelector( EditorSelectors.panels.pageSettings.wrapper );
```
Comment (dmaziar) [inline tests/playwright/pages/editor-page.ts:528]: same here,  please use waitFor()

File: tests/playwright/selectors/editor-selectors.ts [hunk 5]
```diff
@@ -22,6 +19,32 @@ const EditorSelectors = {
 	item: '.elementor-repeater-row-item-title',
 	addNewItem: 'button.elementor-button elementor-repeater-add',
 	plusIcon: '.eicon-plus-circle',
+	topBar: {
+		wrapper: '#elementor-editor-wrapper-v2',
+	},
+	panels: {
```
Comment (dmaziar) [inline tests/playwright/selectors/editor-selectors.ts:25]: nice!

**Review by dmaziar — APPROVED** [review]

**Review by ronenboxer — APPROVED** [review]

**Pull request merged**